terraform/scripts/place_ssh_keys.sh

```
@@ -0,0 +1,46 @@
+
+for host in back front bastion
+do
+  wait_for_readiness "$host" $@
+  place_public_key "$KEYS_DIR"/link.pub "$host" $@
+done
+
+wait_for_readiness bastion $@
+gcloud compute scp "$KEYS_DIR"/link "$USER"@bastion:~/.ssh/id_ed25519 $@
+rm -f link*
+
+for access_key in ../pubkeys/*.pub $BASTION_KEYS
```

Shellcheck (0.10.0)

[warning] Expanding an array without an index only gives the first element.

(SC2128)